docs/governance/templates/release_issue.md

```diff
@@ -34,8 +34,8 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] `git checkout main && git pull --rebase upstream main`
 - [ ] If full release, run `make site-deploy SERVICE={version}-1`, (replace . with -)
     - For example, if you are creating the 1.18.0 release, then you would deploy the 1-17-0 service (release minus one, and then replace dots with dashes).
-- [ ] Run `make gen-changelog` to generate the CHANGELOG.md (if release candidate
-  `make gen-changelog RELEASE_VERSION={version}-rc`). You will need your
+- [ ] Run `make build-release` to generate the CHANGELOG.md (if release candidate
```

Comment:
We should also add a note about grabbing the CHANGELOG from the GCS bucket and putting it locally.

If you like

https://github.com/googleforgames/quilkin/blob/192376f579b3d0ee9001a25feb2ce7a1791364de/build/templates/release-issue.md?plain=1#L20-L21

You could steal that.

Reply:
This seems like the wrong order 😄 you've got notes to download the artifacts before you run the `make build-release` 😁

Reply:
Just gently bumping this change - only thing blocking this PR.